### ISSUE TITLE
No way to set choice screen background

### ISSUE THREAD

**Issue by dvorapa** [LanarsInc/direct-select-flutter#6]
I could not find a way to set a background of the choice screen. It seems it does not inherit a background of the MaterialApp like:

```
ThemeData(
        brightness: Brightness.dark,
        scaffoldBackgroundColor: background,
        backgroundColor: background,
),
```

so even though the whole material app background is set to dark, the Direct Select Flutter Widget uses white somehow.

**Comment by ivanYatcuba** [comment]
Just fixed it in 1.0.4 .  Now choice screen background is the same as scaffoldBackgroundColor

**Comment by iyatsouba** [comment]
@dvorapa  Just fixed it in 1.0.4 . Now choice screen background is the same as scaffoldBackgroundColor

**Comment by dvorapa** [comment]
+1 Thank you!
